rebaser.go

```diff
@@ -24,6 +24,20 @@ func (r *Rebaser) Rebase(
 		return errors.Wrap(err, "get image metadata")
 	}
 
+	workingImageStack, err := metadata.GetStackMetadata(workingImage)
+	if err != nil {
+		return errors.Wrap(err, "get working image stack")
+	}
+
+	newBaseImageStack, err := metadata.GetStackMetadata(newBaseImage)
+	if err != nil {
+		return errors.Wrap(err, "get new base image stack")
```

Comment:
@ashwin-venkatesh I don't think this handles the empty label case. Unhappy path tests around empty labels should drive that out.